core/src/main/java/tachyon/client/TachyonFS.java

```diff
@@ -193,7 +196,8 @@ private TachyonFS(TachyonConf tachyonConf) throws IOException {
   /**
    * Update the latest block access time on the worker.
    * 
-   * @param blockId the local block's id
+   * @param blockId
+   *          the local block's id
```

Comment:
@rvesse Your Eclipse auto-formatter seems out of date. Here is the latest one: https://github.com/amplab/tachyon/blob/master/docs/resources/tachyon-code-formatter-eclipse.xml